### PR TITLE
[W6.1e][W11-B2]Lai Qi Wei

### PR DESCRIPTION
**Pull request by qwlai**
Ready for review.

### CODE REVIEW

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 1]
```diff
@@ -36,6 +35,36 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    @Test
+    public void isAnyNull() throws Exception {
```
Comment (okkhoy) [inline test/java/seedu/addressbook/common/UtilsTest.java:39]: This is a correctly written test. 
Please also be aware of the 3 part test name format that may be useful when you write tests for your project.

**Review by okkhoy — COMMENTED** [review]
Please update the title of the PR to reflect your phase B team name.

**Comment by okkhoy** [comment]
@qwlai 
Some comments added. Please ack & close the PR after reading comments.

**Comment by qwlai** [comment]
Noted with thanks.

**Pull request closed without merging**